packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/waiter-state-machine.ts

```diff
@@ -0,0 +1,82 @@
+// The contents of this file were adapted from the AWS CDK provider framework.
+// https://github.com/aws/aws-cdk/blob/c52ff08cfd1515d35feb93bcba34a3231a94985c/packages/aws-cdk-lib/custom-resources/lib/provider-framework/runtime/cfn-response.ts
```

Comment:
Is this the right link?
```suggestion
// https://github.com/aws/aws-cdk/blob/c52ff08cfd1515d35feb93bcba34a3231a94985c/packages/aws-cdk-lib/custom-resources/lib/provider-framework/waiter-state-machine.ts
```